Makefile

```diff
@@ -124,11 +124,29 @@ test-containerized: run-etcd run-k8s-apiserver build-containerized dist/host-loc
 	-e PLUGIN=calico \
 	-e CNI_SPEC_VERSION=$(CNI_SPEC_VERSION) \
 	-v $(CURDIR):/go/src/github.com/projectcalico/cni-plugin:rw \
+	-v $(CURDIR)/dist:/opt/cni/bin \
 	$(CALICO_BUILD) sh -c '\
 			cd  /go/src/github.com/projectcalico/cni-plugin && \
 			ginkgo'
 	make stop-etcd
 
+.PHONY: k8s-install/scripts/install_cni.test
+## We pre-build the test binary so that we can run it outside a container and allow it
```

Comment:
I don't think these should be `##` - that indicates the text should be emitted when printing `make help`

Reply:
Ah, didn't know that, will fix.